jumpscale/packages/farmmanagement/frontend/store/farmmanagement.js

```diff
@@ -22,12 +22,16 @@ export default {
   },
   actions: {
     getTfgridUrl: async context => {
-      var response = await tfService.getExplorer();
-      var url = JSON.parse(response.data).data.url
-      if (!url.startsWith('http')) {
-        url = `https://${url}/api/v1`;
```

Comment:
you can just use this endpoint to get the explorer url `/admin/actors/identity/get_explorer_url` on development branch
result: 
```
"{\"url\": \"https://explorer.testnet.grid.tf/api/v1\"}"
```

Reply:
@m-motawea this is already happening right now: 
https://github.com/threefoldtech/js-sdk/blob/42e1e1c93b0b3b9992c4b4cb746689abb2cb6b64/jumpscale/packages/admin/actors/admin.py#L34
https://github.com/threefoldtech/js-sdk/blob/42e1e1c93b0b3b9992c4b4cb746689abb2cb6b64/jumpscale/packages/farmmanagement/frontend/services/farmmanagementServices.js#L5

Reply:
This just returns `explorer.tesnet.grid.tf` or `explorer.grid.tf` it does not return what comes after that eg: `/api/v1` in the case of testnet and `/explorer` in the case of mainnet....

Reply:
sorry seems I commented on a wrong line. you don't need to hardcode "/explorer" or "/api/v1". the above actor returns the correct one as you can see in the response

Reply:
@m-motawea no it does not. it return : `"{\"data\": {\"type\": \"testnet\", \"url\": \"explorer.testnet.grid.tf\"}}"`

Reply:
![image](https://user-images.githubusercontent.com/32568986/91857638-701e2880-ec68-11ea-9932-0eefe17f4468.png)

this is the latest on development.